common/lib/xmodule/xmodule/seq_module.py

```diff
@@ -177,9 +177,16 @@ def handle_ajax(self, dispatch, data):  # TODO: bounds checking
         raise NotFoundError('Unexpected dispatch type')
 
     def student_view(self, context):
+        display_items = self.get_display_items()
+
         # If we're rendering this sequence, but no position is set yet,
+        # or exceeds the length of the displayable items,
         # default the position to the first element
-        if self.position is None:
+        if context.get('requested_child') == 'first':
+            self.position = 1
+        elif context.get('requested_child') == 'last':
+            self.position = len(display_items) or None
+        elif self.position is None or self.position > len(display_items):
```

Comment:
The first child is treated as the defacto default in the code, so I kept it as that.  

A case where self.position exceeds the length of the children is an edge case, but one that I thought to safeguard against.  It can happen if the number of children have changed since the section was last accessed by the user.  In that case, it's unclear that we should revert to the last child.  Rather, it seemed safer to revert to the behavior of when the position was unset.

Theoretically, there's still a fundamental flaw in the case where children have been edited since the user last accessed the section.  That is, even if the number of children stays constant, the children themselves could have changed, and position N may no longer refer to the same child.